lib/tip_tap/html_renderable.rb

```diff
@@ -59,7 +59,9 @@ def to_html
     end
 
     def html_attributes
-      {style: inline_styles, class: html_class_name}.reject { |key, value| value.blank? }
+      attributes = {style: inline_styles, class: html_class_name}.reject { |key, value| value.blank? }
+      attributes.merge!({"id" => attrs["id"], "data-toc-id" => attrs["data-toc-id"]}) if self.is_a?(TipTap::Nodes::Heading)
```

Comment:
I think I would rather we override this in the `Heading` node rather than including it here.